core/handlers.py

```diff
@@ -107,6 +107,44 @@
     )
 
 
+@dp.message_handler(commands=["peek"], state="*")
+async def peek_command_handler(msg: types.Message):
+    for job in scheduler.get_jobs():
+        extra = 1 if job.id.endswith("day_before") else 0
+        job_chat_id = job.args[0]
+        if job_chat_id == msg.from_user.id:
+            remainder_is_scheduled_text = _("remainder_is_scheduled")
+            text = f'{remainder_is_scheduled_text} {(job.next_run_time + datetime.timedelta(days=extra)).strftime("%d.%m.%Y (%A)")}'
```

Comment:
`/peek` returns the date of a cleaning, not a remainder

Reply:
hmm, then I understood the meaning of the command wrongly. But if I have reminders for several campuses? It will return date for the earliest one, but I won't understand there is no info about campus number in the message. 

What's the purpose of `/peek` if we have `/schedule` now?

Reply:
Makes sence, I will get rid of `/peek` then